### ISSUE TITLE
Create a release process for the chrome extension

### ISSUE THREAD

**Issue by forgetso** [polkadot-js/extension#1283]
The [chrome extension](https://chrome.google.com/webstore/detail/polkadot%7Bjs%7D-extension/mopnmbcafieddcagagdcbnhejhlodfdd) has not been released since June 2022. It is possible to [auto publish the extension using a simple Gist and Github CI](https://gokatz.me/blog/automate-your-chrome-extension-deployment-in-minutes/). This would probably help out people who are [waiting on functionality](https://github.com/polkadot-js/extension/issues/1268#issuecomment-1620886024) that is already merged into the extension-dapp package.

**Comment by jacogr** [comment]
It is not ready for release, hence no re4lease has been made.

Some things that spring to mind -

1. It doesn't have support for the new metadata formats as of yet
2. It does not support the new Ledger raw signing functionality
3. It has underlying issues with the signer payloads (certain optional fields)

It will be published when it is ready.

**Comment by forgetso** [comment]
Thanks for clarifying that the current release is not finished. I appreciate all the work you do on PolkadotJS.

**Comment by TarikGul** [comment]
Since release cadence is normal again - closing.
